src/Data/CosmosDb/010-queryViews_LiveVacancy_Insert_DefaultField_ApplicationMethod.js

```diff
@@ -1,55 +1,59 @@
-print("Start adding/updating LiveVacancy queryViews with default applicationMethod.");
-
-var query = {
-        "type": "LiveVacancy",
-        "applicationMethod": { $exists: false }
-    },
-    batchUpdateLimit = 500,
-    passThrough = 1;
-
-var maxLoops = Math.ceil(db.queryViews.find().count(query) / batchUpdateLimit);
-
-if (maxLoops === 0) {
-    maxLoops = 1;
-}
-
-do {
-    var matchedDocs = db.queryViews.aggregate([
-        {
-            $match: query
-        },
-        {
-            $sort: { "lastUpdated": 1 }
+{
```

Comment:
Do we need these braces? Should this be an IIFE? I don't understand where the entry point is, I think I'm missing something.

Reply:
I wanted to be a ES6 hipster like Ozzy Scott and use `let` and `const`. Of course Mongoshell is not browser based JS or server side the same way as Node is, so unfortunately we can't use ES6 modules or AMD/Require so the `load` from documentMigration.js is like one context and `var` declarations get hoisted into global scope. I was iffy about an *IIFE* because of the way it would get declared and called so I have used `{` to introduce a scope. the documentMigration.js is the sort of entry point of where the scripts will be invoked from.

Reply:
Not really happy about all this scripting or the complexity of these scripts but this is all we have at the moment. We should move to stored procedures once we have the partition keys sorted.